src/colors.css

```diff
@@ -58,4 +58,69 @@ html[data-color-scheme="dark"] {
   color-scheme: dark;
 }
 
-/** TODO: Add colors for high contrast modes */
+html[data-color-scheme="light-high-contrast"], html:not([data-color-scheme]) {
+  --color-neutral05: #fff;
+  --color-neutral10: #fff;
+  --color-neutral15: #fff;
+  --color-neutral20: #000;
+  --color-neutral25: #000;
+  --color-neutral30: #000;
+  --color-neutral40: #000;
+  --color-neutral50: #000;
+  --color-neutral60: #000;
+  --color-neutral70: #000;
+  --color-neutral80: #000;
+  --color-neutral90: #000;
+
+  --color-danger0: #a5102d;
+  --color-danger1: #a5102d;
+  --color-danger2: #a5102d;
+  --color-danger4: #a5102d;
+  --color-danger5: #a5102d;
+
+  --color-accent8: #000099;
+  --color-accent7: #000099;
+  --color-accent6: #000099;
+  --color-accent5: #000099;
+  --color-accent4: #000099;
+
+  --color-focus: #000099;
+  color-scheme: light-high-contrast;
```

Comment:
That's not a valid property: https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme

This should just be `light`. This is here only to instruct the browser how to design built-in elements.